From ad5e78c618192a58b79a5050d64b18fbad939475 Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Wed, 2 Aug 2023 17:48:27 +0100
Subject: [PATCH 2/3] setup.py: don't generate version info in build

this has already been generated by the time conda get's involved
---
 setup.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 5961e4f4c..5f1e35799 100755
--- a/setup.py
+++ b/setup.py
@@ -91,6 +91,7 @@ def get_version_info():
 
     # If this is a pycbc git repo always populate version information using GIT
     try:
+        raise RuntimeError("conda-forge feedstock git repo")
         vinfo = _version_helper.generate_git_version_info()
     except:
         vinfo = vdummy()
@@ -125,8 +126,9 @@ Repository status is {vinfo.status}\"\"\"
 
 from pycbc._version import *
 """
-    with open('pycbc/version.py', 'wb') as f:
-        f.write(version_script.encode('utf-8'))
+    # disable overwriting during conda-forge build
+    #with open('pycbc/version.py', 'wb') as f:
+    #    f.write(version_script.encode('utf-8'))
 
     from pycbc import version
     return version.version
-- 
2.39.5

